x-pack/libbeat/tests/system/test_management.py

```diff
@@ -85,8 +85,9 @@ def test_enroll_bad_pw(self):
         new_content = open(config_path, 'r').read()
         assert config_content == new_content
 
-    @unittest.skipIf(not INTEGRATION_TESTS,
-                     "integration tests are disabled, run with INTEGRATION_TESTS=1 to enable them.")
+    # @unittest.skipIf(not INTEGRATION_TESTS,
+    #                  "integration tests are disabled, run with INTEGRATION_TESTS=1 to enable them.")
```

Comment:
Do these 2 lines need to be commented out?

Reply:
they will be back when the skip is removed